src/components/prompt/prompt.jsx

```diff
@@ -31,29 +47,64 @@ const PromptComponent = props => (
             <Box>
                 <input
                     autoFocus
-                    className={styles.input}
+                    className={styles.variableNameTextInput}
                     placeholder={props.placeholder}
                     onChange={props.onChange}
                     onKeyPress={props.onKeyPress}
                 />
             </Box>
-            <Box className={props.showMoreOptions ? styles.moreOptions : styles.hideMoreOptions}>
-                <ComingSoonTooltip
-                    className={styles.moreOptionsAccordion}
-                    place="right"
-                    tooltipId="variable-options-accordion"
-                >
-                    <div className={styles.moreOptionsText}>
-                        <FormattedMessage
-                            {...messages.moreOptionsMessage}
-                        />
-                        <img
-                            className={styles.moreOptionsIcon}
-                            src={dropdownIcon}
-                        />
-                    </div>
-                </ComingSoonTooltip>
-            </Box>
+            {props.showMoreOptions ?
+                <div>
+                    {props.isStage ?
+                        <div className={styles.infoMessage}>
+                            <FormattedMessage
+                                {...messages.availableToAllSpritesMessage}
+                            />
+                        </div> :
+                        <Box className={styles.optionsRow}>
+                            <label>
+                                <input
+                                    checked
```

Comment:
If you change this to `defaultChecked` then that bug you saw with needing to click it twice will be fixed